merlin/table/tensorflow_column.py

```diff
@@ -46,6 +46,10 @@ class TensorflowColumn(TensorColumn):
     A SeriesLike column backed by Tensorflow tensors
     """
 
+    @classmethod
+    def _transpose(cls, values):
```

Comment:
Does this method need to be called somewhere as part of the rest of this change?

Reply:
I think this is probably left over from an earlier iteration of the code